R/tags.R

```diff
@@ -907,10 +975,11 @@ as.tags <- function(x, ...) {
 
 #' @export
 as.tags.default <- function(x, ...) {
-  if (is.list(x) && !isTagList(x))
-    unclass(x)
-  else
+  if (is.list(x)) {
```

Comment:
There's an `as.tags.list` method now, so when would we hit this code path?

Reply:
In this case: `as.tags(structure(list(), class = "foo"))`

I'll make sure to add a test that we handle this properly

Reply:
```suggestion
  # Plain (non-classed) lists will hit as.tags.list(), but lists with a class
  # will get here. (tagLists will already have been handled by
  # as.tags.shiny.tag.list)
  if (is.list(x)) {
```